web/src/pages/TransactionRunDetail/Content.tsx

```diff
@@ -27,7 +27,7 @@ const Content = () => {
           <S.SectionLeft>
             <Typography style={{fontSize: 16, marginBottom: 24}}>Execution steps</Typography>
 
-            {transaction?.steps.map(({name, version, ...test}, index) => {
+            {/* {transaction?.steps.map(({name, version, ...test}, index) => {
```

Comment:
Commenting this for now. I think this should be part of the Transaction Run model instead of Transaction.